terraform/eks/otlp/shared.tf

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------------------
```

Comment:
i assume shared.tf will not only be used for config file, but all the shared resources?

Reply:
Changed the name to `oltp.tf` to avoid confusion. Basically it's the file that is used for both `pull mode` and `push mode`, sth. like the `main.tf` in a module, but I couldn't think of a good name. Any suggestion is also welcomed.